server/src/main/java/org/apache/kafka/server/config/Defaults.java

```diff
@@ -0,0 +1,278 @@
+    public static final String INTER_BROKER_PROTOCOL_VERSION = MetadataVersion.latestProduction().version();
+
+    /** ********* Controlled shutdown configuration *********/
+    public static final int CONTROLLED_SHUTDOWN_MAX_RETRIES = 3;
+    public static final int CONTROLLED_SHUTDOWN_RETRY_BACKOFF_MS = 5000;
+    public static final boolean CONTROLLED_SHUTDOWN_ENABLE = true;
+
+    /** ********* Group coordinator configuration *********/
+    public static final int GROUP_MIN_SESSION_TIMEOUT_MS = 6000;
+    public static final int GROUP_MAX_SESSION_TIMEOUT_MS = 1800000;
+    public static final int GROUP_INITIAL_REBALANCE_DELAY_MS = 3000;
+    public static final int GROUP_MAX_SIZE = Integer.MAX_VALUE;
+
+    /** ********* New group coordinator configs *********/
+    public static final boolean NEW_GROUP_COORDINATOR_ENABLE = false;
+    public static final List<String> GROUP_COORDINATOR_REBALANCE_PROTOCOLS = Arrays.asList(Group.GroupType.CLASSIC.toString());
+    public static final int GROUP_COORDINATOR_NUM_THREADS = 1;
+
+    /** ********* Consumer group configs *********/
+    public static final int CONSUMER_GROUP_SESSION_TIMEOUT_MS = 45000;
+    public static final int CONSUMER_GROUP_MIN_SESSION_TIMEOUT_MS = 45000;
+    public static final int CONSUMER_GROUP_MAX_SESSION_TIMEOUT_MS = 60000;
+    public static final int CONSUMER_GROUP_HEARTBEAT_INTERVAL_MS = 5000;
+    public static final int CONSUMER_GROUP_MIN_HEARTBEAT_INTERVAL_MS = 5000;
+    public static final int CONSUMER_GROUP_MAX_HEARTBEAT_INTERVAL_MS = 15000;
+    public static final int CONSUMER_GROUP_MAX_SIZE = Integer.MAX_VALUE;
+    public static final List<String> CONSUMER_GROUP_ASSIGNORS = Collections.singletonList(RangeAssignor.class.getName());
+
+    /** ********* Offset management configuration *********/
+    public static final int OFFSET_METADATA_MAX_SIZE = OffsetConfig.DEFAULT_MAX_METADATA_SIZE;
+    public static final int OFFSETS_LOAD_BUFFER_SIZE = OffsetConfig.DEFAULT_LOAD_BUFFER_SIZE;
+    public static final short OFFSETS_TOPIC_REPLICATION_FACTOR = OffsetConfig.DEFAULT_OFFSETS_TOPIC_REPLICATION_FACTOR;
+    public static final int OFFSETS_TOPIC_PARTITIONS = OffsetConfig.DEFAULT_OFFSETS_TOPIC_NUM_PARTITIONS;
+    public static final int OFFSETS_TOPIC_SEGMENT_BYTES = OffsetConfig.DEFAULT_OFFSETS_TOPIC_SEGMENT_BYTES;
+    public static final int OFFSETS_TOPIC_COMPRESSION_CODEC = OffsetConfig.DEFAULT_OFFSETS_TOPIC_COMPRESSION_TYPE.id;
+    public static final int OFFSETS_RETENTION_MINUTES = 7 * 24 * 60;
+    public static final long OFFSETS_RETENTION_CHECK_INTERVAL_MS = OffsetConfig.DEFAULT_OFFSETS_RETENTION_CHECK_INTERVAL_MS;
+    public static final int OFFSET_COMMIT_TIMEOUT_MS = OffsetConfig.DEFAULT_OFFSET_COMMIT_TIMEOUT_MS;
+    public static final short OFFSET_COMMIT_REQUIRED_ACKS = OffsetConfig.DEFAULT_OFFSET_COMMIT_REQUIRED_ACKS;
+
+    /** T********* ransaction management configuration *********/
```

Comment:
It looks like something went wrong on this line

Reply:
Same on a few other comments below

Reply:
fixed